src/test/java/com/wikia/webdriver/elements/communities/desktop/components/articlepreview/MobilePreviewModal.java

```diff
@@ -1,61 +1,59 @@
 package com.wikia.webdriver.elements.communities.desktop.components.articlepreview;
 
-import com.wikia.webdriver.common.logging.Log;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 
-import org.openqa.selenium.By;
+/*
+  Mobile preview modal. Entry point for that preview could be find in the article's editor on Desktop.
+  User is able to see how his/her page would look on mobile view.
+ */
 
 public class MobilePreviewModal extends WikiBasePageObject {
 
   public MobilePreviewModal() {
     super();
   }
 
-  public MobilePreviewModal heroImageIsPresent() {
-    waitForElementToBePresent(".has-hero-image", "Hero image");
+  @FindBy(css = ".has-hero-image")
+  private WebElement heroImage;
 
-    return this;
-  }
-
-  public MobilePreviewModal infoboxIsPresent() {
-    waitForElementToBePresent(".portable-infobox", "Portable infobox");
+  @FindBy(css = ".portable-infobox")
+  private WebElement portableInfobox;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)